src/test/kotlin/com/teamdev/jxbrowser/gradle/JxBrowserPluginFunctionalTest.kt

```diff
@@ -94,7 +94,10 @@ internal class JxBrowserPluginFunctionalTest {
                 "jxbrowser-mac-$jxBrowserVersion.jar",
                 "jxbrowser-mac-arm-$jxBrowserVersion.jar",
             )
-
+        if (jxBrowserVersion.startsWith("8")) {
```

Comment:
I don't think it's necessary to test that here. We can cover it in the test below, which specifically tests the JxBrowser 8 jars.